README.md

```diff
@@ -1 +1,41 @@
-# hw01
\ No newline at end of file
+# hw01
+
+
+
+**Who am I?**
+========
+
+<img src="https://github.com/recalderamirez/hw01/blob/master/LUIS.jpg?raw=true" width="400">
+
+
+Hello!
+
+My name is _**Luis Recalde**_. I am from Paraguay. 
+
+I am a second MPP _(Master of Public Policy)_ student at the **Harris School of Public Policy.** My area of policy interest is International Development, broadly defined. During the Spring quarter I will be in Chicago, practicing self-quarantine (due to the COVID-19 pandemic) while "attending" and studying for my classes.
+
+### **My favorite classes** #
+
+In my two years studying a master's degree at **the University of Chicago** my favorite classes were:
```

Comment:
He added personal and professional info, which I appreciate. Also, I think his use of bullets here was very adequate.

Reply:
I think that the bio contains all the requested elements in homework. Luis used them in a correct way and, in terms of content, his bio has the expected information for a summarized bio.